bigframes/session/__init__.py

```diff
@@ -307,6 +309,72 @@ def read_gbq(
                 api_name="read_gbq",
             )
 
+    def _filters_to_query(self, query_or_table, filters):
+        """Convert filters to query"""
+
+        if (filters is None) or (len(filters) == 0):
+            return query_or_table
+
+        valid_operators = ["IN", "NOT IN", "=", ">", "<", ">=", "<=", "!="]
+
+        sub_query = (
+            f"({query_or_table})" if _is_query(query_or_table) else query_or_table
+        )
+
+        where_clause = ""
+        if filters:
+            if not isinstance(filters, list):
+                raise ValueError("Filters should be a list.")
```

Comment:
Don't need to check this. Any Iterable should be fine.

Reply:
Done.